### PR TITLE
added Dockerfile and docker-compose.yml file for faculty-v3

### PR DESCRIPTION
**Pull request by ebm17915**
# DESCRIPTION
```
added Dockerfile and docker-compose.yml file for faculty-v3
View Dockerfile for dependencies/packages
```
# REQUIREMENTS & DEPENDENCIES
```
Have the docker-compose.yml file in faculty-v3 directory
Have testing .env from web-demo-php7 server
```
# TESTING 

1. Clone media repository: git clone https://github.com/csun-metalab/faculty-v3.git
2. Obtain testing .env from web-demo-php7 server
3. Import faculty-v3's docker-compose.yml file if it isn't merged yet
4. Run docker-compose up -d
5. View in localhost:8080

# NOTES
1. Any other necessary packages/dependencies?
2. Errors/bugs that can be fixed or looked into? Aside from media images.
3. Etc.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+FROM csunmetalab/environment:base
+
+# Add Yarn Debian Repo
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
+ && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
+ && apt-get update \
+# Update and install additional dependencies/packages
+  && apt-get update && apt-get install -y \
+  php7.2 \
```
Comment (LisaSmith1) [inline Dockerfile]: php7.2 is installed on the base image

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  wget \
```
Comment (LisaSmith1) [inline Dockerfile]: Maybe wget should be added to the base image?

**Comment by ruizjorgealt** [comment]
Tested, and it works great 👍

**Review by ruizjorgealt — APPROVED** [review]
Good job!

**Review by matthewfritz — APPROVED** [review]
If Jorge says it's good, then it's good.

**Pull request merged**